sql/backfill.go

```diff
@@ -188,18 +193,44 @@ func (sc *SchemaChanger) getTableSpan() (roachpb.Span, error) {
 	}, nil
 }
 
+// backoffDurationDuringBackfill is the approximate time that the backfill
+// suspends activity between steps of the backfill. This value was picked
+// through some experimentation:
+// 1. Run block-writer against a 3 node cluster on GCE.
+// 2. Run create/drop index, create/drop column schema changes.
+var backoffDurationDuringBackfill = 75 * time.Millisecond
```

Comment:
It would be nice if this could adjust itself dynamically. For example, if after each chunk we sleep for the duration of that chunk, we'd automatically use half our time for schema changes (and could easily adjust the ratio up or down).